src/js/components/ServicesTable.js

```diff
@@ -219,26 +219,28 @@ var ServicesTable = React.createClass({
   },
 
   getDestroyConfirmDialog: function () {
-    let message = null;
     const {state} = this;
     let service = state.serviceToChange;
+    let serviceName = '';
     let itemText = 'Service';
     if (service instanceof ServiceTree) {
       itemText = 'Group';
     }
 
     if (service) {
-      message = (
-        <div className="container-pod flush-top container-pod-short-bottom">
-          <h2 className="text-danger text-align-center flush-top">Destroy {itemText}</h2>
-          <p>
-            Are you sure you want to destroy <span className="emphasize">{service.getId()}</span>? This action is irreversible.
-          </p>
-          {this.getErrorMessage()}
-        </div>
-      );
+      serviceName = service.getId();
     }
 
+    let message = (
```

Comment:
honestly you're right. i was just going the way it was being done at that point. i'll make the change right now